### ISSUE TITLE
Lack of option "Mode" (Fixed or fit to text)in button element in RU version of editor

### ISSUE THREAD

**Issue by agnieszkakowalska** [EDMdesigner/editor-issues#94]
Most probably this is another issue with too long translations:

![obraz](https://user-images.githubusercontent.com/17157918/30022552-1089a502-916c-11e7-83bc-8908c68803f1.png)

**Comment by smiska** [comment]
Yes, indeed. The translations are for the mode toggle button: "Фиксировано" , "Подогнать под текст". Can you please provide a shorter version, so we can fix it for you? Thanks.

**Comment by agnieszkakowalska** [comment]
Sure. We don't know what is the max number of chars, but please see below:

> fit to text - "По размеру текста" or "По тексту" or "Подогнать"
> fixed - "Фиксировано" must be short enough but we can shorten it to "Фикс."
> The longer, the more understandable, of course

**Comment by smiska** [comment]
Does this seem all rigth?

![razmer](https://user-images.githubusercontent.com/16736043/30152092-4e65a4f0-93b2-11e7-9276-420b114fd2dd.png)

**Comment by agnieszkakowalska** [comment]
Yes! Thank you :)

**Comment by smiska** [comment]
I made the modification, it will be released soon with the new editor version. Please look out for the dashboard notification.

**Comment by agnieszkakowalska** [comment]
Thank you @smiska!

**Comment by smiska** [comment]
Hi @agnieszkakowalska , I'm not sure if I notified you already or not, but if you change to editor version r20170926 this fix will be available.

**Comment by agnieszkakowalska** [comment]
Thank you :)
